test/unit/splitter_test.rb

```diff
@@ -40,6 +40,10 @@ class CssSplitterTest < ActiveSupport::TestCase
     assert_equal ["a{foo:bar;}", "@media print{b{baz:qux;}", "c{quux:corge;}", "}", "d{grault:garply;}"], CssSplitter::Splitter.split_string_into_rules(has_media)
   end
 
+  test "#split_string_into_rules containing @keyframes" do
```

Comment:
I think a spec that is acutally calling `#split_string_into_rules` directly would be better here.  the spec would have a higher documenation value and it would be more consistent with the other specs.